gender_novels/common.py

```diff
@@ -152,21 +152,25 @@ def load_file_remotely(file_path, current_file_type):
             # has \r and \n -> replace with only \n
             return text.replace('\r\n', '\n')
 
-def get_text_file_encoding(filepath):
+
+def get_encoding_type(filepath):
```

Comment:
What's the argument for changing get_text_file_encoding back to get_encoding_type? I had changed the name to make it obvious that this function only works with text files. I am also ok with get_encoding_type but in that case it should also be changed in corpus_gen.py

Reply:
That seems to be a mistaken merge. I didn’t change that. Will revert.

Reply:
Ah -- I had fixed the doctest for the older function and that overwrote the changes.  Fixing for new name.